src/mame/heathkit/tlb.cpp

```diff
@@ -709,6 +720,41 @@ static INPUT_PORTS_START( watz19 )
 
 INPUT_PORTS_END
 
+static INPUT_PORTS_START( gp19 )
+	PORT_INCLUDE( tlb )
+
+	PORT_MODIFY("SW402")
+	PORT_DIPNAME( 0x80, 0x00, "Automatic Holdscreen")    PORT_DIPLOCATION("SW402:8")
+	PORT_DIPSETTING(    0x00, "Disabled (VT100 mode)")
+	PORT_DIPSETTING(    0x80, "Enabled (Z19 mode)")
+
+	PORT_START("SW1")
+	PORT_DIPNAME( 0x03, 0x01, "Trailing characters for Tektronix message")   PORT_DIPLOCATION("SW1:1,2")
+	PORT_DIPSETTING(    0x00, "none")
+	PORT_DIPSETTING(    0x01, "CR")
+	PORT_DIPSETTING(    0x02, "none")
+	PORT_DIPSETTING(    0x03, "CR,EOT")
+	PORT_DIPNAME( 0x04, 0x01, "Shift key")                                   PORT_DIPLOCATION("SW1:3")
+	PORT_DIPSETTING(    0x00, "Normal")
+	PORT_DIPSETTING(    0x04, "Shift key inverts CAPS LOCK")
+	PORT_DIPNAME( 0x08, 0x00, "Terminal Transmission Rate")                  PORT_DIPLOCATION("SW1:4")
+	PORT_DIPSETTING(    0x00, "Only limited by baud rate")
+	PORT_DIPSETTING(    0x08, "One character per 16.7 mSec")
```

Comment:
Should Baud be capitalised, since it’s an abbreviation of a name (Baudot), and hence a proper noun?  The first letter of “Hertz” is always capitalised.

Reply:
I wasn't sure, so googled it, and should not be capitalized according to wikipedia and a few other sites if it's the full word, only capitalized if it is abbreviated. It also said the same thing with hertz.

Reply:
If you followed that line of reasoning, then Volts, Ohms, Ampheres and Watts should always start with a capital letter, together with any other unit of measurement that's named after a person. Not sure if kilowatt-hour should be kiloWatt-hour though. I would think that normal usage would dictate lower-case.

Reply:
Yes baud is with a small letter, and while you're at it mSec would be msec ;)